packages/graphiql/src/api/actions/sessionActions.ts

```diff
@@ -8,6 +8,7 @@ export enum SessionActionTypes {
   OperationSucceeded = 'OperationSucceeded',
   OperationErrored = 'OperationErrored',
   TabChanged = 'TabChanged',
+  ParametersChanged = 'ParametersChanged',
```

Comment:
Think `SearchParamatersChanged` or `QueryStringParametersChanged` would be clearer. A parameter is quite a general term.